pyproject.toml

```diff
@@ -268,6 +268,20 @@ directory = "misc"
 name = "Miscellaneous internal changes"
 showcontent = true
 
+[tool.coverage.html]
+show_contexts = true
+skip_covered = false
+
+[tool.coverage.paths]
+_site-packages-to-src-mapping = [
+  "src",
+  "*/src",
+  '*\src',
```

Comment:
Why does it have both forward and backwards slash?

Reply:
also one below this also has a backwards slash

Reply:
Probably not, but I saw this practice a lot in the wild. I think modern coveragepy normalizes them but I don't want to take a chance.

Reply:
Probably not what?

Reply:
Oh, my brain parsed it as “is this necessary?” :laughing: Apologies.

These are basically explicit Windows path variants.

Reply:
Ah, that makes a lot of sense.

Reply:
Having recently skimmed a bunch of codecov docs I'm pretty sure there's enough path normalization mentioned in a bunch of places that this shouldn't be needed.

Reply:
@jakkdl this is not Codecov but coveragepy.